src/pages/haeriRoh/Main/MainHr.js

```diff
@@ -1,7 +1,159 @@
+
+  // 2. comment inputvalue add
+
+  // 3. comment state => <div>~~~~</div>
+
+  return (
+    <div>
+      <div className="insta">
+        <div className="container">
+          <Nav />
+          <main>
+            <div className="feeds">
+              <div className="container">
+                <article>
+                  <div className="feedBox">
+                    <div className="topFeed">
+                      <div className="idBox">
+                        <div className="profilePic" />
+                        <span className="instaId">yesharryy</span>
+                      </div>
+                      <div className="more">
+                        <i className="fa-solid fa-ellipsis" />
+                      </div>
+                    </div>
+                    <div className="midFeed" />
+                    <div className="bottomFeed">
+                      <div className="iconBottom">
+                        <div className="rightIconBottom">
+                          <i className="fa-solid fa-heart" />
+                          <i className="fa-regular fa-comment" />
+                          <i className="fa-regular fa-share-from-square" />
+                        </div>
+                        <div className="leftIconBottom">
+                          <i className="fa-regular fa-bookmark" />
+                        </div>
+                      </div>
+                      <div className="likeBottom">
+                        <div className="likeBox">
+                          <div className="likePic" />
+                          <span className="likeCount">
+                            <b>yesharryy</b>님 외 <b>10명</b>이 좋아합니다
+                          </span>
+                        </div>
+                      </div>
+                      <div className="commentBox">
+                        <div className="myComment">
+                          <b>yesharryy</b> 김우주 보고싶어~!!
+                        </div>
+                        <div className="otherComment">
+                          {/* <p id="commentsList" /> */}
+                          {/* {comment.map(function (a) {
+                            return <li>{a}</li>;
+                          })} */}
+                          <Comment comment2={comment} />
+                          <input
+                            type="text"
+                            className="msg"
+                            placeholder=" 댓글 달기..."
+                            required
+                            value={inputValue} //공백
+                            onChange={e => {
+                              setInputValue(e.target.value);
+                            }}
+                          />
+                          <button
+                            onClick={() => {
+                              let copy = [...comment];
+                              copy.push(inputValue);
+                              setComment(copy);
```

Comment:
```suggestion
                              setComment([...comment, inputValue]);
```
이렇게 한줄로 작성할 수도 있을 것 같습니다.